packages/web-twig/src/Resources/components/TextFieldBase/README.md

```diff
@@ -64,14 +64,17 @@ Without lexer:
 | `class`             | `string`                                                             | `null`  | no       | Custom CSS class                                                                                                                   |
 | `hasPasswordToggle` | `bool`                                                               | `false` | no       | If true, the `type` is set to `password` and a password toggle is shown                                                            |
 | `helperText`        | `string`                                                             | `null`  | no       | Custom helper text                                                                                                                 |
+| `UNSAFE_helperText` | `string`                                                             | `null`  | no       | Unescaped custom helper text                                                                                                       |
 | `id`                | `string`                                                             | —       | yes      | Input and label identification                                                                                                     |
 | `isDisabled`        | `bool`                                                               | `false` | no       | If true, input is disabled                                                                                                         |
 | `isFluid`           | `bool`                                                               | `false` | no       | If true, the element spans to the full width of its parent                                                                         |
 | `isLabelHidden`     | `bool`                                                               | `false` | no       | If true, label is hidden                                                                                                           |
 | `isMultiline`       | `bool`                                                               | `false` | no       | If true, rendered DOM element is `textarea`                                                                                        |
 | `isRequired`        | `bool`                                                               | `false` | no       | If true, input is required                                                                                                         |
 | `label`             | `string`                                                             | —       | yes      | Label text                                                                                                                         |
+| `UNSAFE_label`      | `string`                                                             | —       | yes      | Unescaped label text                                                                                                               |
```

Comment:
...this is just a reminder to previous comment (this should not be required).

Reply:
The label is required by this component. I have made a note.